mscore/startupWizard.cpp

```diff
@@ -121,10 +121,23 @@ void StartupWizard::autoSelectShortcuts(QString keyboardLayout, QString os)
       layoutToShortcut->insert("Sebeolsik (세벌식)", ":/data/shortcuts.xml");
       layoutToShortcut->insert("OTHER", ":/data/shortcuts.xml");
     
+      QString os;
+#if defined(Q_OS_MAC)
+      os = "mac";
+#elif defined(Q_OS_WIN)
+      os = "win";
+#elif defined(Q_OS_LINUX)
+      os = "linux";
+#else
+      os = "other";
+#endif
+    
       QString fileLocation = layoutToShortcut->value(keyboardLayout);
       if (os == "mac") {
             fileLocation = ":/data/shortcuts_MAC.xml";
       }
+      QSettings s;
+      s.setValue("osIdentified", true);
```

Comment:
This doesn't seem needed to me. MuseScore knows exactly what OS it is running on, this isn't decided at run time